src/testing/cluster.zig

```diff
@@ -358,10 +358,10 @@ pub fn ClusterType(comptime StateMachineType: anytype) type {
         pub fn restart_replica(cluster: *Self, replica_index: u8) !void {
             assert(cluster.replica_health[replica_index] == .down);
 
-            var nonce = cluster.replicas[replica_index].nonce + 1;
+            const nonce = cluster.replicas[replica_index].nonce + 1;
             // Pass the old replica's Time through to the new replica. It will continue to tick while
             // the replica is crashed, to ensure the clocks don't desynchronize too far to recover.
-            var time = cluster.replicas[replica_index].time;
+            const time = cluster.replicas[replica_index].time;
```

Comment:
This one is intentionally `var`, to force the compiler to make a copy here and not do a pass-by-reference

Reply:
We need to express this in the code (with a dummy mutation) otherwise, the compiler will catch it as an error when we upgrade to Zig 0.12